src/inflators/image.ts

```diff
@@ -0,0 +1,46 @@
+import { addComponent } from "bitecs";
+import { MediaImage } from "../bit-components";
+import { addObject3DComponent } from "../utils/jsx-entity";
+import { AlphaMode, create360ImageMesh, createImageMesh } from "../utils/create-image-mesh";
+import { ProjectionMode } from "../utils/projection-mode";
+import { Texture } from "three";
+import { HubsWorld } from "../app";
+import { EntityID } from "../utils/networking-types";
+
+export interface ImageParams {
+  cacheKey: string;
+  texture: Texture;
+  ratio: number;
+  projection: ProjectionMode;
+  alphaMode: AlphaMode;
+  alphaCutoff: number;
+}
+
+export function inflateImage(world: HubsWorld, eid: EntityID, params: ImageParams) {
+  const { texture, ratio, projection, alphaMode, alphaCutoff, cacheKey } = params;
+  const mesh =
+    projection === ProjectionMode.SPHERE_EQUIRECTANGULAR
+      ? create360ImageMesh(texture, alphaMode, alphaCutoff)
+      : createImageMesh(texture, ratio, alphaMode, alphaCutoff);
+  addObject3DComponent(world, eid, mesh);
+  addComponent(world, MediaImage, eid);
+
+  if (projection !== undefined) {
+    MediaImage.projection[eid] = projection;
+  } else {
+    MediaImage.projection[eid] = ProjectionMode.FLAT;
+  }
+  if (alphaMode !== undefined) {
+    MediaImage.alphaMode[eid] = alphaMode;
+  } else {
+    MediaImage.alphaMode[eid] = AlphaMode.Opaque;
+  }
+  if (alphaCutoff !== undefined) {
+    MediaImage.alphaCutoff[eid] = alphaCutoff;
+  } else {
+    MediaImage.alphaCutoff[eid] = 0;
+  }
```

Comment:
Not a blocking issue but a comment for future clean up: Here can be similified to use a common pattern as [other inflators](https://github.com/mozilla/hubs/blob/master/src/inflators/ambient-light.ts) do

```typescript
export type XXXParams = {
  foo?: string;
  bar?: number;
};

const DEFAULTS: Required<XXXParams> = {
  foo: 1.0,
  bar: 'default'
};

function xxxSystem(world: HubsWorld, eid: EntityID, params: XXXParams) {
  const requiredParams = Object.assign({}, DEFAULTS, params) as Required<XXXParams>;
  addComponent(world, XXX, eid);
  XXX.foo[eid] = requiredParams.foo;
  XXX.bar[eid] = requiredParams.bar;
  ...
}
```